packages/react/src/themes/teams/components/Chat/chatMessageStyles.ts

```diff
@@ -1,10 +1,19 @@
 import { ComponentSlotStylesInput, ICSSInJSStyle } from '../../../types'
-import { ChatMessageProps } from '../../../../components/Chat/ChatMessage'
+import {
+  default as ChatMessage,
+  ChatMessageProps,
+  ChatMessageState,
+} from '../../../../components/Chat/ChatMessage'
 import { ChatMessageVariables } from './chatMessageVariables'
 import { screenReaderContainerStyles } from '../../../../lib/accessibility/Styles/accessibilityStyles'
 import { pxToRem } from '../../../../lib'
 
-const chatMessageStyles: ComponentSlotStylesInput<ChatMessageProps, ChatMessageVariables> = {
+const chatMessageActionsClassNameSelector = `& .${ChatMessage.slotClassNames.actionMenu}`
```

Comment:
this seems to be used in the single place - lets just inline this expression there, to aid readability

Reply:
Done 👍